src/test/kotlin/lotto/domain/LottoCompanyTest.kt

```diff
@@ -0,0 +1,57 @@
+package lotto.domain
+
+import lotto.model.LottoTicket
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+/**
+ * Created by Jaesungchi on 2022.05.25..
+ */
+class LottoCompanyTest {
+    @ParameterizedTest
+    @ValueSource(strings = ["1, 2, 3, 4, 5, 6"])
+    internal fun `지난 주 당첨 번호 입력 받은걸 ','로 나눈다`(source: String) {
+        assertThat(LottoCompany.of(source).winningTicket).isNotNull
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = ["1, 2, 3, 4, 5"])
+    internal fun `지난 주 당첨 번호는 6개가 아니라면 IllegalArgumentException을 던진다`(source: String) {
+        assertThrows<IllegalArgumentException> {
+            LottoCompany.of(source)
+        }
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = ["1, 2, 3, 4, 5, 6"])
+    internal fun `로또와 당첨번호가 일치하는 갯수를 찾고 몇등인지 잘 반환한다`(source: String) {
```

Comment:
```suggestion
    internal fun `로또번호와 당첨번호가 모두 일치하는 1등을 반환한다`(source: String) {
```

몇 등이라는 것은 꼭 코드를 읽어봐야 이게 몇 등이 나오는지에 대해 알 수 있다고 봐요.

테스트를 통하여, 1등인지, 2등인지, 3등인지에 대해 좀 더 명확한 테스트를 해주셨으면 합니다!